index.ios.js

```diff
@@ -1,27 +1,4 @@
-import React, { Component } from 'react';
-import {
-  AppRegistry,
-  StatusBar,
-  View,
-} from 'react-native';
-import { theme } from './config';
+import { AppRegistry } from 'react-native';
```

Comment:
This file now is the same than index.android.js, should we have just a single index.js? Or the current react version will look for different entry points?